src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,92 @@
+package duke.task;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates a list of tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructs a new {@code TaskList} with no tasks.
```

Comment:
I really like how this class is very succinct and has very detailed JavaDoc. Your code looks really neat here